hack/verify-vulnerabilities.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Wondering if we should rename this to `hack/verify-security.sh` to match the github workflow 🤔

Reply:
Yeah I was thinking the same. Sounds a bit strange but probably better for consistency